docs/features-3.6.md

```diff
@@ -0,0 +1,77 @@
+* [#12674](https://github.com/argoproj/argo-workflows/pull/12674): CronWorkflows and WorkflowTemplates now display their title and descriptions in the list view.
+* [#12199](https://github.com/argoproj/argo-workflows/pull/12199): You can specify HTTP headers to use to detect IP addresses using the `IP_KEY_FUNC_HEADERS` environment variable. This is used in the rate limiter.
+* [#13695](https://github.com/argoproj/argo-workflows/pull/13695): You can now retry a single node from a workflow, even if the workflow succeeded.
+* [#13610](https://github.com/argoproj/argo-workflows/pull/13610): You can now filter with prefixes and patterns in the workflow list.
+
+## Metrics
+
+* [#13265](https://github.com/argoproj/argo-workflows/pull/13265): The workflow controller can now emit metrics over OpenTelemetry GRPC protocol
+    * [#13267](https://github.com/argoproj/argo-workflows/pull/13267): with selectable temporality
+    * [#13268](https://github.com/argoproj/argo-workflows/pull/13268): configuration of what is emitted
+* Many of the metrics have been updated which will require you [to change how you use them](upgrading.md#metrics_changes) and there are some new ones:
+    * [#13269](https://github.com/argoproj/argo-workflows/pull/13269): Version information in the controller
+    * [#13270](https://github.com/argoproj/argo-workflows/pull/13270): Is this controller the leader
+    * [#13271](https://github.com/argoproj/argo-workflows/pull/13271): Kubernetes API calls duration
+    * [#13272](https://github.com/argoproj/argo-workflows/pull/13272): Pod phase monitoring
+    * [#13274](https://github.com/argoproj/argo-workflows/pull/13274): CronWorkflows counters
+    * [#13497](https://github.com/argoproj/argo-workflows/pull/13497): CronWorkflows policy counters
+    * [#13275](https://github.com/argoproj/argo-workflows/pull/13275): Workflow Template counters
+    * [#13735](https://github.com/argoproj/argo-workflows/pull/13735): Counters to check if you're using deprecated features
+* [#11927](https://github.com/argoproj/argo-workflows/pull/11927): There is a new `retries` variable available in metrics describing the number of retries.
+* [#11857](https://github.com/argoproj/argo-workflows/pull/11857): Pod missing metrics will be emitted before pods are created
+
+## Controller
+
+* [#13358](https://github.com/argoproj/argo-workflows/pull/13358): You can use multiple mutexes and semaphores in the same workflow or template, and use both type of lock at the same time
+* [#13419](https://github.com/argoproj/argo-workflows/pull/13419): The controller uses a queue when archiving workflows to improve memory management when archiving a large number of workflows at once
+* [#12441](https://github.com/argoproj/argo-workflows/pull/12441): Plugins can now be stopped, so that a stopped workflow will shutdown its plugin nodes
+* The OSS artifact driver:
```

Comment:
"Plugins" and "Artifacts" sections might be good to split out so it's not just "Controller". Artifacts and Plugins also run in the Server + Executor and Agent, respectively, so not in the Controller

Reply:
I've renamed it General. Most people don't care which component a feature happens in, it's an irrelevance to them.

Reply:
I don't think that's necessarily accurate -- if I use artifacts, having an "Artifacts" section would make it easy for me to see what's new with that "feature set".
Permissions are also relevant per component. But usage wise, it's less about the component and more about the broader "feature set" (or "area" as we use in labels) like "Artifacts", "Plugins", "SSO", etc, which is what I suggested above